### ISSUE TITLE
Cannot create new entries in collections

### ISSUE THREAD

**Issue by trevorstinson** [statamic/v2-hub#1782]
### Expected behaviour
Should be able to create and edit entries in collections using the control panel.

### Actual behaviour
When I click "Save" on a new entry, or when editing an entry, I receive an error message stating `There was a problem saving the data. Please check your logs.` This is reproducible with all of my existing Statamic installs (on my Mac and on a live server) as well as in brand new installations using the built-in sample content.

### Steps to reproduce
1.  Click "Create Entry" or "Edit" on an entry.
2.  Fill in fields, make changes, etc.
3.  Click "Save"

### Server Details

**Operating System**: Mac and Ubuntu

**Web Server:** nginx (default Laravel Valet setup on Mac and default Forge setup on Digital Ocean)

**PHP Version:** 7.2

**Statamic Version:** 2.8.4

**Updated from an older Statamic or fresh install:** Updated from 2.8.3 or a fresh install.

**List of installed addons:** None.

### Logs

<details>

```
[2018-02-17 05:02:23] production.ERROR: ErrorException: mb_strpos() expects parameter 1 to be string, array given in /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Support/Str.php:69
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'mb_strpos() exp...', '/Users/trevor/S...', 69, Array)
#1 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Support/Str.php(69): mb_strpos(Array, 'mimes')
#2 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Support/helpers.php(597): Illuminate\Support\Str::contains(Array, 'mimes')
#3 /Users/trevor/Sites/bug/statamic/core/CP/Publish/Publisher.php(226): str_contains(Array, 'mimes')
#4 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Support/Collection.php(135): Statamic\CP\Publish\Publisher->Statamic\CP\Publish\{closure}(Array, 'extra.datetime')
#5 /Users/trevor/Sites/bug/statamic/core/CP/Publish/Publisher.php(246): Illuminate\Support\Collection->each(Object(Closure))
#6 /Users/trevor/Sites/bug/statamic/core/CP/Publish/Publisher.php(206): Statamic\CP\Publish\Publisher->preparedRequest(Array)
#7 /Users/trevor/Sites/bug/statamic/core/CP/Publish/EntryPublisher.php(53): Statamic\CP\Publish\Publisher->validate(Array, Array, Array)
#8 /Users/trevor/Sites/bug/statamic/core/CP/Publish/Publisher.php(83): Statamic\CP\Publish\EntryPublisher->initialValidation()
#9 /Users/trevor/Sites/bug/statamic/core/CP/Publish/EntryPublisher.php(25): Statamic\CP\Publish\Publisher->publish()
#10 /Users/trevor/Sites/bug/statamic/core/Http/Controllers/PublishController.php(106): Statamic\CP\Publish\EntryPublisher->publish()
#11 [internal function]: Statamic\Http\Controllers\PublishController->save(Object(Illuminate\Http\Request))
#12 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(256): call_user_func_array(Array, Array)
#13 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(164): Illuminate\Routing\Controller->callAction('save', Array)
#14 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(112): Illuminate\Routing\ControllerDispatcher->call(Object(Statamic\Http\Controllers\PublishEntryController), Object(Illuminate\Routing\Route), 'save')
#15 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#16 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(114): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#20 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(68): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(Statamic\Http\Controllers\PublishEntryController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'save')
#21 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Routing/Route.php(203): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'Statamic\\Http\\C...', 'save')
#22 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Routing/Route.php(134): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#23 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Routing/Router.php(708): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 /Users/trevor/Sites/bug/statamic/core/Http/Middleware/CP/Authenticate.php(62): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 [internal function]: Statamic\Http\Middleware\CP\Authenticate->handle(Object(Illuminate\Http\Request), Object(Closure))
#28 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#29 /Users/trevor/Sites/bug/statamic/core/Http/Middleware/Outpost.php(48): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#30 [internal function]: Statamic\Http\Middleware\Outpost->handle(Object(Illuminate\Http\Request), Object(Closure))
#31 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#32 /Users/trevor/Sites/bug/statamic/core/Http/Middleware/CP/Localize.php(25): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#33 [internal function]: Statamic\Http\Middleware\CP\Localize->handle(Object(Illuminate\Http\Request), Object(Closure))
#34 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#35 /Users/trevor/Sites/bug/statamic/core/Http/Middleware/CP/DefaultLocale.php(28): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#36 [internal function]: Statamic\Http\Middleware\CP\DefaultLocale->handle(Object(Illuminate\Http\Request), Object(Closure))
#37 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#38 /Users/trevor/Sites/bug/statamic/core/Http/Middleware/CpEnabled.php(23): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#39 [internal function]: Statamic\Http\Middleware\CpEnabled->handle(Object(Illuminate\Http\Request), Object(Closure))
#40 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#41 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#42 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#43 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Routing/Router.php(710): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#44 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Routing/Router.php(674): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#45 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Routing/Router.php(635): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#46 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(236): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#47 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#48 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#49 /Users/trevor/Sites/bug/statamic/core/Http/Middleware/PersistStache.php(22): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#50 [internal function]: Statamic\Http\Middleware\PersistStache->handle(Object(Illuminate\Http\Request), Object(Closure))
#51 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#52 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php(50): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#53 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#54 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#55 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#56 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#57 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#58 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(62): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#59 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#60 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#61 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#62 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#63 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#64 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(59): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#65 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#66 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#67 /Users/trevor/Sites/bug/statamic/core/StaticCaching/Middleware/Retrieve.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#68 [internal function]: Statamic\StaticCaching\Middleware\Retrieve->handle(Object(Illuminate\Http\Request), Object(Closure))
#69 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#70 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#71 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#72 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#73 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#74 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#75 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(122): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#76 /Users/trevor/Sites/bug/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(87): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#77 /Users/trevor/Sites/bug/index.php(155): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#78 /Users/trevor/.composer/vendor/laravel/valet/server.php(133): require('/Users/trevor/S...')
#79 {main} 
```

</details>

**Comment by ebeauchamps** [comment]
Did you check the permission on your Content folder?
then, can you put right here the top of your log file?

**Comment by jackmcdade** [comment]
Docs on proper permissions here: https://docs.statamic.com/installing#permissions

**Comment by trevorstinson** [comment]
Bug is still present after re-upgrading, clearing the cache, and ensuring the permissions are correct. Here is the latest stack trace.

<details>

```[2018-02-17 13:27:52] production.ERROR: ErrorException: mb_strpos() expects parameter 1 to be string, array given in /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Support/Str.php:69
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'mb_strpos() exp...', '/Users/trevor/S...', 69, Array)
#1 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Support/Str.php(69): mb_strpos(Array, 'mimes')
#2 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Support/helpers.php(597): Illuminate\Support\Str::contains(Array, 'mimes')
#3 /Users/trevor/Sites/trevorstinson.com/statamic/core/CP/Publish/Publisher.php(226): str_contains(Array, 'mimes')
#4 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Support/Collection.php(135): Statamic\CP\Publish\Publisher->Statamic\CP\Publish\{closure}(Array, 'extra.datetime')
#5 /Users/trevor/Sites/trevorstinson.com/statamic/core/CP/Publish/Publisher.php(246): Illuminate\Support\Collection->each(Object(Closure))
#6 /Users/trevor/Sites/trevorstinson.com/statamic/core/CP/Publish/Publisher.php(206): Statamic\CP\Publish\Publisher->preparedRequest(Array)
#7 /Users/trevor/Sites/trevorstinson.com/statamic/core/CP/Publish/EntryPublisher.php(53): Statamic\CP\Publish\Publisher->validate(Array, Array, Array)
#8 /Users/trevor/Sites/trevorstinson.com/statamic/core/CP/Publish/Publisher.php(83): Statamic\CP\Publish\EntryPublisher->initialValidation()
#9 /Users/trevor/Sites/trevorstinson.com/statamic/core/CP/Publish/EntryPublisher.php(25): Statamic\CP\Publish\Publisher->publish()
#10 /Users/trevor/Sites/trevorstinson.com/statamic/core/Http/Controllers/PublishController.php(106): Statamic\CP\Publish\EntryPublisher->publish()
#11 [internal function]: Statamic\Http\Controllers\PublishController->save(Object(Illuminate\Http\Request))
#12 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(256): call_user_func_array(Array, Array)
#13 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(164): Illuminate\Routing\Controller->callAction('save', Array)
#14 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(112): Illuminate\Routing\ControllerDispatcher->call(Object(Statamic\Http\Controllers\PublishEntryController), Object(Illuminate\Routing\Route), 'save')
#15 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#16 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(114): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#20 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(68): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(Statamic\Http\Controllers\PublishEntryController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'save')
#21 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Routing/Route.php(203): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'Statamic\\Http\\C...', 'save')
#22 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Routing/Route.php(134): Illuminate\Routing\Route->runWithCustomDispatcher(Object(Illuminate\Http\Request))
#23 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Routing/Router.php(708): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 /Users/trevor/Sites/trevorstinson.com/statamic/core/Http/Middleware/CP/Authenticate.php(62): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#27 [internal function]: Statamic\Http\Middleware\CP\Authenticate->handle(Object(Illuminate\Http\Request), Object(Closure))
#28 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#29 /Users/trevor/Sites/trevorstinson.com/statamic/core/Http/Middleware/Outpost.php(48): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#30 [internal function]: Statamic\Http\Middleware\Outpost->handle(Object(Illuminate\Http\Request), Object(Closure))
#31 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#32 /Users/trevor/Sites/trevorstinson.com/statamic/core/Http/Middleware/CP/Localize.php(25): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#33 [internal function]: Statamic\Http\Middleware\CP\Localize->handle(Object(Illuminate\Http\Request), Object(Closure))
#34 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#35 /Users/trevor/Sites/trevorstinson.com/statamic/core/Http/Middleware/CP/DefaultLocale.php(28): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#36 [internal function]: Statamic\Http\Middleware\CP\DefaultLocale->handle(Object(Illuminate\Http\Request), Object(Closure))
#37 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#38 /Users/trevor/Sites/trevorstinson.com/statamic/core/Http/Middleware/CpEnabled.php(23): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#39 [internal function]: Statamic\Http\Middleware\CpEnabled->handle(Object(Illuminate\Http\Request), Object(Closure))
#40 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#41 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#42 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#43 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Routing/Router.php(710): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#44 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Routing/Router.php(674): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#45 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Routing/Router.php(635): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#46 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(236): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#47 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#48 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(139): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#49 /Users/trevor/Sites/trevorstinson.com/statamic/core/Http/Middleware/PersistStache.php(22): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#50 [internal function]: Statamic\Http\Middleware\PersistStache->handle(Object(Illuminate\Http\Request), Object(Closure))
#51 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#52 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php(50): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#53 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#54 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#55 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#56 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#57 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#58 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(62): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#59 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#60 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#61 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#62 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#63 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#64 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(59): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#65 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#66 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#67 /Users/trevor/Sites/trevorstinson.com/statamic/core/StaticCaching/Middleware/Retrieve.php(33): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#68 [internal function]: Statamic\StaticCaching\Middleware\Retrieve->handle(Object(Illuminate\Http\Request), Object(Closure))
#69 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#70 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#71 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#72 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#73 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#74 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#75 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(122): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#76 /Users/trevor/Sites/trevorstinson.com/statamic/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(87): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#77 /Users/trevor/Sites/trevorstinson.com/index.php(155): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#78 /Users/trevor/.composer/vendor/laravel/valet/server.php(133): require('/Users/trevor/S...')
#79 {main}  
```

</details>

**Comment by edalzell** [comment]
What's the actual error (should be above the stack trace) @trevorstinson?

**Comment by trevorstinson** [comment]
@edalzell I made a mistake in my markdown that caused the error to be hidden. I've edited the comment above so that you can see the error.

**Comment by jackmcdade** [comment]
I can recreate it too. Working on a fix right now.

**Comment by jackmcdade** [comment]
Fixed with [2.8.5](https://statamic.com/changelog#2.8.5). Thanks for the report!
